common/selectors/transaction/scheduleTimestamp.ts

```diff
@@ -0,0 +1,34 @@
+import { AppState } from 'reducers';
+import { getScheduleTimestamp } from './fields';
+import moment from 'moment';
+import { EAC_SCHEDULING_CONFIG } from 'libs/scheduling';
+
+interface ICurrentScheduleTimestamp {
+  raw: string;
+  value: Date | null;
+}
+
+const isValidCurrentScheduleTimestamp = (state: AppState) => {
+  const currentScheduleTimestamp = getScheduleTimestamp(state);
+  const raw = currentScheduleTimestamp.raw;
+  const value = currentScheduleTimestamp.value;
+
+  const fiveMinFromNow = moment()
+    .add(5, 'm')
+    .toDate();
+
+  if (!value || value <= fiveMinFromNow || !isCorrectFormat(raw)) {
```

Comment:
This could be one line:

`return value && value > fiveMinFromNow && isCorrectFormat(raw);`